docs/dev/stream/operators/process_function.md

```diff
@@ -322,3 +322,26 @@ ctx.timerService.registerEventTimeTimer(coalescedTime)
 {% endhighlight %}
 </div>
 </div>
+
+### Fault Tolerance
+
+Timers registered within `ProcessFunction` are fault tolerant.
+
+Timers registered within `ProcessFunction` will be checkpointed by Flink. Upon restoring, timers that are checkpointed
+from the previous job will be restored on whatever new instance is responsible for that key.
+
+#### Processing Time Timers
+
+For processing timer timers, note that the firing time of a timer is an absolute value of when to fire.
+
+What this means is that if a checkpointed timer’s firing processing timestamp is t (which is basically the registering
```

Comment:
> (which is basically the registering time + configured trigger time)

This is often the case, but not necessarily true. Esp. for processing time, the timer can also be set to something completely different. I'd remove this to avoid confusion.